phpcs.xml

```diff
@@ -1,11 +1,15 @@
 <?xml version="1.0"?>
 <ruleset name="Performance-Module">
     <rule ref="Newfold"/>
-    <config name="testVersion" value="7.3-"/>
-    <config name="minimum_supported_wp_version" value="6.3"/>
+    <config name="testVersion" value="7.0-"/>
+    <config name="minimum_supported_wp_version" value="5.8"/>
+    <exclude-pattern>./assets</exclude-pattern>
+    <exclude-pattern>./build</exclude-pattern>
     <rule ref="WordPress-Core">
         <exclude name="WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid"/>
         <exclude name="WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase"/>
+        <exclude name="WordPress.NamingConventions.ValidVariableName.PropertyNotSnakeCase"/>
+        <exclude name="WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase"/>
```

Comment:
@bradp we should not exclude these as this would not match the new standards followed by all our other modules and diverge from them as we add more code.

I think we can spend some time here to clean up any old camel cased code in the module. What do you think?